packages/lodestar-validator/src/api/interface/events.ts

```diff
@@ -1,6 +1,58 @@
+import {EventEmitter} from "events";
+import StrictEventEmitter from "strict-event-emitter-types";
+import {BlockEventPayload, ChainHead, ChainReorg} from "@chainsafe/lodestar-types";
 import {IStoppableEventIterable} from "@chainsafe/lodestar-utils";
-import {BeaconEvent, BeaconEventType} from "../impl/rest/events/types";
+
+export enum BeaconEventType {
+  BLOCK = "block",
+  CHAIN_REORG = "chain_reorg",
+  HEAD = "head",
+  CLOCK_SLOT = "clock_slot",
+  CLOCK_EPOCH = "clock_epoch",
```

Comment:
I think we should try to upstream those first, before adding it here. I also have personal interested in this since we would like lodestar-validator to work with lighthouse in Chainguardian.

Reply:
These events are not being sent via the API. A local clock is reusing the event emitter to emit clock events.
Definitely would be interested to get the clock exposed in the standard API tho